PhysicsTools/NanoAOD/python/jetsAK8_cff.py

```diff
@@ -122,6 +122,7 @@
         particleNet_XttVsQCD = Var("bDiscriminator('pfParticleNetFromMiniAODAK8DiscriminatorsJetTags:HttvsQCD')",float,doc="ParticleNet X->tau_h tau_h vs. QCD score: Xtt/(Xtt+QCD)",precision=10),
         particleNet_XtmVsQCD = Var("bDiscriminator('pfParticleNetFromMiniAODAK8DiscriminatorsJetTags:HtmvsQCD')",float,doc="ParticleNet X->mu tau_h vs. QCD score: Xtm/(Xtm+QCD)",precision=10),
         particleNet_XteVsQCD = Var("bDiscriminator('pfParticleNetFromMiniAODAK8DiscriminatorsJetTags:HtevsQCD')",float,doc="ParticleNet X->e tau_h vs. QCD score: Xte/(Xte+QCD)",precision=10),
+        particleNet_WVsQCD = Var("bDiscriminator('pfParticleNetFromMiniAODAK8DiscriminatorsJetTags:WvsQCD')",float,doc="ParticleNet W->qq vs. QCD score: Xqq+Xcc/(Xqq+Xcc+QCD)",precision=10),
```

Comment:
This will only work when reading new MiniAODs that already have `pfParticleNetFromMiniAODAK8DiscriminatorsJetTags:WvsQCD` stored. For running NANO on existing MiniAODs, the `particleNet_WVsQCD` branch will get a `-1000` because the new discriminant was not available. Is that the intended behavior?

Reply:
Yes, it is intended. This is for NanoV15, where Run-3 and Run-2 samples will be reMini (with `15_X`, correct me if I am wrong), so there will be no backport to `14_0_X`.